sdk/core/azure-core-amqp/src/impl/rust_amqp/amqp/private/claims_based_security_impl.hpp

```diff
@@ -46,14 +43,11 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
         CbsTokenType type,
         std::string const& audience,
         std::string const& token,
+        Azure::DateTime const& tokenExpirationTime,
```

Comment:
should this be the last element in the param list and have a default value? not saying to change it , just asking .

Reply:
As I mentioned, @antkmsft made an excellent suggestion that for an `_internal` type, it makes sense to not default the `Context` parameter. 

For public APIs we default `Context` but for internal APIs, they usually exist in service to a public API, and as such, you want to force the consumer of the API taking the context to make a choice about which context to provide to the method call.